release-team/role-handbooks/release-signal/project-boards.md

```diff
@@ -0,0 +1,150 @@
+- `Reporting Status`: **SHOULD BE REMOVED IMO, JUST UNNECESSARILY COMPLICATING STUFF**
+
+#### Setting up the CI Signal Project Board
+
+- Copy the prior project board as it has some settings related to Workflows, fields, grouping, etc that help with some basic
+  automation and the look and feel (you can look at the past Workflows in prior tabs).
+
+<img alt="How to Duplicate View" height="428" src="img/project-board-duplicate-view.png" width="400"/>
+
+- You need to add a new option to the field `K8s Release` to add and view items being tracked in the current release.
+
+<img alt="How to add a new option to K8s Release field" height="406" src="img/project-board-add-new-k8s-release.png" width="800"/>
+
+- Your new board will need to have the filter updated (see red line below) to make sure it is pulling in issues tagged with the
+  proper release.
+
+<img alt="How to Update Filter" height="400" src="img/project-board-update-filter.png" width="464"/>
+
+### Bug Triage Project Board
+
+The Bug Triage component of the team is using a [GitHub project board](https://github.com/orgs/kubernetes/projects/80) to track
+the current status of all issues and PRs targeting the release, their priorities, and to distribute the work amongst the team.
+
+Release Signal team members are expected to assign themselves as the `Responsible` person to track an issue to the extent of their
+time availability throughout the cycle. The release signal team should keep an eye on the board for new issues/PRs and make sure
+they are tracked.
+
+The project board is public so that both the release team and anyone interested from the community can stay updated about the
+current status of issues and PRs that are targetting the release. Write access to the board is limited to members
+of [@kubernetes/release-team-bug-triage](https://github.com/orgs/kubernetes/teams/release-team-bug-triage).
+
+New issues and PRs that target the current release milestone are automatically added to the board with
+a [a prow job](https://prow.k8s.io/?job=periodic-sync-bug-triage-github-project-*). The prow job is defined
+in [kubernetes/test-infra](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml)
+and the script can be found
+in [kubernetes/sig-release](https://github.com/kubernetes/sig-release/blob/master/release-team/hack/sync-bug-triage-github-project-beta.sh).
+
+#### Information on the Bug Triage Board
+
+For each board item, the following details are set and managed by the release signal team:
+
+- `Responsible`: The release signal team member that is responsible for tracking the progress on the issue/PR. This is not the
+  same as the issue assignee.
+- `Notes`: Short note by the release signal team member regarding the issue/PR,
+  e.g. `[6/3] No progress made since last comment; notified them about the code freeze`
```

Comment:
If a tracking issue is not updated for a long period like 2 weeks. Should we bring it to the v1.xx release meeting?

Could we have some simple bars?

Reply:
These are the board views. They do not contain anything about reporting in general. Can you please check the `Reporting` section in readme? Feel free to add a review if something is not clear in that.